.github/scripts/integrate-rc-to-develop.sh

```diff
@@ -0,0 +1,111 @@
+    git fetch origin
+    git checkout "origin/$RC_BRANCH"
+    git checkout -b "$MERGE_BRANCH"
+
+    echo "Merging develop into $MERGE_BRANCH to detect conflicts"
+    if git merge origin/develop --no-commit --no-ff; then
+        echo "Clean merge - no conflicts detected"
+        git commit -m "Merge develop into $MERGE_BRANCH"
+    else 
+        echo "Merge conflicts detected"
+        if git status --porcelain | grep -q "package.json"; then 
+            echo "Conflicts in package.json detected, auto-resolving"
+            git checkout --theirs package.json
+            git add package.json
+        fi 
+
+        if git status --porcelain | grep -q "^UU\|^AA\|^DD"; then
+            git commit -m "Merge develop into $MERGE_BRANCH (manual resolution needed)"
+        else
+            git commit -m "Merge develop into $MERGE_BRANCH (auto-resolved)"
+        fi
+    fi
+
+    git push -u origin "$MERGE_BRANCH"
+    echo "Created merge branch $MERGE_BRANCH"
+}
+
+create_pull_request() {
+    local RC_BRANCH=$1
+    local MERGE_BRANCH=$2
+    
+    local pr_title="Merge $RC_BRANCH to develop - fresh commit on $TODAY"
+    local pr_body="**Details:**
+    - Fresh commit detected on: $DATE_DISPLAY
+    - Source: This branch is based on $RC_BRANCH
+
+This PR was created automatically by the daily RC integration workflow.
+⚠️ **Note**: Any merge conflicts will be visible in this PR and need manual resolution."
+
+    echo "Creating pull request for $MERGE_BRANCH -> develop"
+    
+    if gh pr create \
+        --title "$pr_title" \
+        --body "$pr_body" \
+        --base develop \
+        --head "$MERGE_BRANCH"; then
+        echo "PR created successfully for $MERGE_BRANCH"
+    else
+        echo "ERROR: Failed to create PR for $MERGE_BRANCH"
+        return 1
+    fi
+}
+
+# Main execution
+echo "Checking RC branches for fresh commits on $DATE_DISPLAY"
+
+RC_BRANCHES=$(git branch -r | grep -E 'origin/v[0-9.]+-(R|r)(C|c)' | sed 's|.*origin/||')
+
+if [[ -z "$RC_BRANCHES" ]]; then
+    echo "No RC branches found"
+    exit 0
+fi
+
+for RC_BRANCH in $RC_BRANCHES; do 
+    echo "Checking branch: $RC_BRANCH"
+
+    COMMIT_DATE=$(git log -1 --format=%cd --date=format:%m%d "origin/$RC_BRANCH" 2>/dev/null || echo "")
+    
+    if [[ "$COMMIT_DATE" != "$TODAY" ]]; then
+        echo "No fresh commit found on $RC_BRANCH (commit date: $COMMIT_DATE)"
+        continue
+    fi
+
+    echo "Found fresh commit on $RC_BRANCH"
+
+    EXISTING_OPEN_PR=$(gh pr list --base develop --state open --json headRefName,number --jq ".[] | select(.headRefName | startswith(\"merge-${RC_BRANCH}-to-develop-\")) | .number" 2>/dev/null | head -1 || echo "")
+    
+    if [[ -n "$EXISTING_OPEN_PR" ]]; then
+        echo "Open PR already exists for $RC_BRANCH (PR #$EXISTING_OPEN_PR), skipping"
+        continue
+    fi
+    
+    MERGE_BRANCH="merge-${RC_BRANCH}-to-develop-${TODAY}"
+
+    if branch_exists "$MERGE_BRANCH" && [[ -z "$EXISTING_PR" ]]; then
```

Comment:
```suggestion
    if branch_exists "$MERGE_BRANCH" && [[ -z "$EXISTING_OPEN_PR" ]]; then
```
I think it's supposed to be this?

Reply:
Woops good catch. Fixed here: https://github.com/msupply-foundation/open-msupply/pull/8422/commits/f76637439625852bb836089db5e5a7dec093da7e